File: corecomply-mvp/web/src/types/person.ts
export interface Person {
  id: number;
  firstName: string;
  lastName: string;
  email: string;
  phone?: string;
  jobTitle?: string;
  role: string;
  department: string;
  status: string;
  startDate?: string;
  lastLogin?: string;
  avatarUrl?: string;
  createdAt: string;
}

export interface PersonStats {
  total: number;
  active: number;
  avgComplianceScore: number;
  needsAttention: number;
}

File: corecomply-mvp/api/CoreComply.Api/Domain/Entities/Person.cs
namespace CoreComply.Api.Domain.Entities;

public class Person
{
    public int Id { get; set; }
    public string FirstName { get; set; } = string.Empty;
    public string LastName { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string? Phone { get; set; }
    public string? JobTitle { get; set; }
    public string Role { get; set; } = "Reviewer";
    public string Department { get; set; } = string.Empty;
    public string Status { get; set; } = "Active"; // Active, Inactive, OnLeave
    public DateTime? StartDate { get; set; }
    public DateTime? LastLogin { get; set; }
    public string? AvatarUrl { get; set; }
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}
